storm-core/src/jvm/backtype/storm/scheduler/resource/RAS_Node.java

```diff
@@ -72,10 +71,9 @@ public RAS_Node(String nodeId, Set<Integer> allPorts, boolean isAlive,
             _availMemory = this.getTotalMemoryResources();
             _availCPU = this.getTotalCpuResources();
             _slots.addAll(_freeSlots);
-            for (WorkerSlot ws : _slots) {
-                _slotToExecs.put(ws, new ArrayList<ExecutorDetails>());
-            }
         }
+        this._cluster = cluster;
```

Comment:
Can we also not use `this` for member variables that have been tagged as such by naming convention? That's exactly what the underscore is intended to communicate. There's no benefit to using both.